tcms/telemetry/templates/telemetry/testing/test-case-health.html

```diff
@@ -0,0 +1,100 @@
+                        <option value="">----------</option>
+                    </select>
+                </div>
+
+                <label for="id_version" class="col-md-1 col-lg-1">
+                    {% trans "Version" %}
+                </label>
+                <div class="col-md-3">
+                    <select name="version" id="id_version" class="form-control selectpicker">
+                        <option value="">----------</option>
+                    </select>
+                </div>
+
+                <label for="id_build" class="col-md-1 col-lg-1">
+                    {% trans "Build" %}
+                </label>
+                <div class="col-md-3">
+                    <select name="build" id="id_build" class="form-control selectpicker">
+                        <option value="">----------</option>
+                    </select>
+                </div>
+            </div>
+
+            <div class="form-group">
+                <label for="id_test_plan" class="col-md-1 col-lg-1">
+                    {% trans "Test Plan" %}
+                </label>
+                <div class="col-md-3">
+                    <select name="test_plan" id="id_test_plan" class="form-control selectpicker">
+                        <option value="">----------</option>
+                    </select>
+                </div>
+
+                <label class="col-md-1 col-lg-1" for="id_after">{% trans "After" %}</label>
+                <div class="col-md-3 col-lg-3">
+                    <div class="input-group date-time-picker-pf">
+                        <input type="text" class="form-control" id="id_after">
+                        <span class="input-group-addon">
+                            <span class="fa fa-calendar"></span>
+                        </span>
+                    </div>
+
+                    {% include "datetimepicker_script.html" with selector="#id_after" %}
+                </div>
+
+                <label class="col-md-1 col-lg-1" for="id_before">{% trans "Before" %}</label>
+                <div class="col-md-3 col-lg-3">
+                    <div class="input-group date-time-picker-pf">
+                        <input type="text" class="form-control" id="id_before">
+                        <span class="input-group-addon">
+                            <span class="fa fa-calendar"></span>
+                        </span>
+                    </div>
+
+                    {% include "datetimepicker_script.html" with selector="#id_before" %}
+                </div>
+            </div>
+        </form>
+
+        <table class="table table-bordered table-with-horizontal-scroll cell-border" id="table-most-failed">
+            <thead>
+                <tr>
+                    <th class="header" colspan="3"> {% trans "Most Failed Tests" %} </th>
+                </tr>
+                <tr>
+                    <th class="header"> {% trans "Test Case" %}</th>
```

Comment:
Instead of `Test Case` is it possible to either:

1) have these headers only at the top or
2) have the category name in this cell and show an empty colspan=3 row between them ?

Reply:
2. I don't get that. Seems the same as 1.

1. Sure. It even looks better this way IMO :
<img width="1653" alt="image" src="https://user-images.githubusercontent.com/18421997/64979528-96fd1680-d8c0-11e9-95df-d95ecb8cde55.png">

Do you want to leave it this way?